frontend/src/components/VGlobalAudioSection/VGlobalAudioSection.vue

```diff
@@ -3,9 +3,10 @@
     <VGlobalAudioTrack v-if="audio" :audio="audio" />
     <VIconButton
       v-if="audio"
-      class="absolute top-0 z-10 border-none ltr:right-0 rtl:left-0"
+      class="absolute top-0 z-30 border-none ltr:right-0 rtl:left-0"
```

Comment:
30 seems a bit high and a bit arbitrary, I think using named z-indices, or starting a new [stacking context](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Positioning/Understanding_z_index/The_stacking_context) (considering that the parent div has `position: sticky`), might be more elegant.

Reply:
And merged the PR before pushing the changes 🤦